Assignment2.sql

```diff
@@ -0,0 +1,47 @@
+USE baseball;
+
+
+#Calculating the Historic batting average
+create or replace table HistBattingAvg(batter int, batting_average float) ENGINE=MyISAM
+	SELECT batter,(sum(Hit)/sum(atBat)) as batting_average 
+	from batter_counts 
+	group by batter;
+
+#Viewing the results
+select * 
+	from histbattingavg;
+
+#Calculating the Annual Batting Average
+create or replace table AnnualBatAvg(batter int, batting_average float) ENGINE=MyISAM
```

Comment:
You can give name to column YEAR(local_date) and access that in Group clause too.

Reply:
done.